templates/gnark/sindri.json

```diff
@@ -0,0 +1,9 @@
+{
+  "circuit_type": "gnark",
+  "name": "{{ packageName }}",
+  "package_name": "{{ packageName }}",
+  "circuit_struct_name": "Circuit",
```

Comment:
Choosing to make the "circuit_struct_name" hard coded for now.  We could template it, but then the I/O component after `sindri init` is going to have to explain what exactly the "circuit_struct_name" field is...and that feels pretty involved for the type of user that's going to be playing with `sindri init`

Reply:
The default struct name sounds good.

Note that I specifically made the field names here camelCase however because this is the common convention for JSON. We added support for this on the backend, and this is the convention that I think we should use when generating/publishing the JSON schema.

Reply:
I have no opinion on which is better - but could you double check that camelCase is supported (by revising the fields in the template to camelcase, taring the result, and compiling)? Because the only reason I changed these fields is that circuit compile failed initially with the following error.
```
Unable to create a new circuit. status=412 response={'error': 'ForgeInvalidUploadError', 'message': "Invalid sindri.json: [{'loc': ('circuit_struct_name',), 'msg': 'field required', 'type': 'value_error.missing'}, {'loc': ('gnark_version',), 'msg': 'field required', 'type': 'value_error.missing'}, {'loc': ('package_name',), 'msg': 'field required', 'type': 'value_error.missing'}]"
```

Reply:
It should be on Karl's recent changes that are failing on stage now, but I'll dig into this and make sure we support it.